helm/alfresco-content-services/Chart.yaml

```
@@ -31,6 +31,12 @@ dependencies:
     alias: postgresql-syncservice
     condition: >-
       alfresco-sync-service.syncservice.enabled,postgresql-syncservice.enabled
+  - name: common
+    alias: alfresco-admin-app
+    repository: https://activiti.github.io/activiti-cloud-helm-charts
+    version: 7.6.0
+    condition: >-
+      alfresco-content-services.alfresco-admin-app.enabled,alfresco-admin-app.enabled
```

First condition doesn't make sense to me (could also be removed from adw imho)

```suggestion
      alfresco-admin-app.enabled
```

---

Fixed